src/public/curriculum/mothed/mothed.json

```diff
@@ -9,6 +9,118 @@
       "numFormat": ".2~f"
     }
   },
+  "navTabs": {
```

Comment:
Do we have any sense of what is actually appropriate to show for the MothEd project? I'd be interested in seeing a more extensive change, possibly even removal of the nav tabs altogether, as a more interesting test case. But if that's either not appropriate for MothEd or substantially more work it could be put off to another PR.